ND Supporting Document 2_2.adoc

```diff
@@ -2660,80 +2660,118 @@ Remark: Some systems might require the presence of the SAN extension. In this ca
 *FCS_TLSS_EXT.1.2*
 
 [arabic, start=552]
-. The evaluator shall verify that the TSS contains a description of how the TOE technically prevents the use of old SSL and TLS versions.
+. The evaluator shall verify that the TSS contains a description of how the TOE technically prevents the use of unsupported and undefined SSL and TLS versions.
 
 *FCS_TLSS_EXT.1.3*
 
 [arabic, start=553]
-. If using ECDHE or DHE ciphers, the evaluator shall verify that the TSS describes the key agreement parameters of the server Key Exchange message.
+. If using ECDHE or DHE ciphers, the evaluator shall verify that the TSS describes the algorithms and key sizes the TSF supports for authenticating itself to TLS clients. The evaluator shall ensure these algorithms are consistent with the selected ciphersuites.
 
 *FCS_TLSS_EXT.1.4*
 
 [arabic, start=554]
-. The evaluator shall verify that the TSS describes if session resumption based on session IDs is supported (RFC 4346 and/or RFC 5246) and/or if session resumption based on session tickets is supported (RFC 5077).
+. The evaluator shall verify that the TSS describes the key exchange parameters of the server Key Exchange message. The evaluator shall ensure these algorithms are consistent with the selected ciphersuites.
+
+*FCS_TLSS_EXT.1.5*
+
+[arabic, start=555]
+. The evaluator shall verify that the TSS describes if session resumption based on session IDs is supported (RFC 4346 and/or RFC 5246), if session resumption based on session tickets is supported (RFC 5077) and/or if session resumption according to RFC8446 is supported.
 . If session tickets are supported, the evaluator shall verify that the TSS describes that the session tickets are encrypted using symmetric algorithms consistent with FCS_COP.1/DataEncryption. The evaluator shall verify that the TSS identifies the key lengths and algorithms used to protect session tickets.
 . If session tickets are supported, the evaluator shall verify that the TSS describes that session tickets adhere to the structural format provided in section 4 of RFC 5077 and if not, a justification shall be given of the actual session ticket format.
 
+*FCS_TLSS_EXT.1.6*
+
+[arabic, start=556]
+. The evaluator shall verify that TSS describes whether the list of supported ciphersuites can be configured or not.
+
+*FCS_TLSS_EXT.1.7*
```

Comment:
This should be FCS_TLSS_EXT.1.8. FCS_TLSS_EXT.1.8 is listed in NDcPP TLSWGProposal_TLSv13_v13d.docx.  FCS_TLSS_EXT.1.8 does not appear in your list of assigned SFRs, and that may be the reason for the confusion.

Reply:
@plughy2 There is not a FCS_TLSS_EXT.1.8 in the most recent proposal document. FCS_TLSS_EXT.1.7 is correct here.